man/docker-daemon.8.md

```diff
@@ -513,6 +513,16 @@ By default docker will pick up the zfs filesystem where docker graph
 
 Example use: `docker daemon -s zfs --storage-opt zfs.fsname=zroot/docker`
 
+## Btrfs options
+
+#### btrfs.min_free_space
+
+Specifies the size to use when creating the subvolume which is used for
+containers. If user uses disk quota for btrfs, docker will check the
+minimum free space when a new container is created.
```

Comment:
I'm still a bit confused by this description; does this set the *size* of the subvolume (per container? for all containers?) or set the minimum amount of space that should be kept available on disk?